todos-app/src/App/App.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { useState } from "react";
+import NewTodoInput from "./NewTodoInput/NewTodoInput";
+import TodoList from "./TodoList/TodoList";
+import ListFooter from "./ListFooter/ListFooter";
+import { ITodo } from "../DataStructure";
+import setUUID from "../UUID";
+import "../App.css";
+
+const App: React.FC = () => {
```

Comment:
minor, don't really need to specify the return type, it should be inferred by typescript

Reply:
Fixed it